src/MongoDB.Bson/TargetFramework.cs

```diff
@@ -0,0 +1,29 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+namespace MongoDB.Bson
+{
+    internal static class TargetedFramework
+    {
+        public static string Current =>
+#if NET452
+            "net452";
+#elif NETSTANDARD1_5
+            "netstandard15";
+#elif NETSTANDARD2_0
+            "netstandard20";
+#endif
```

Comment:
a default else part is not necessary since any else part(if we already have all branches from the targetFrameworks list) won't even be compiled, so the default branch can contain even something like this: `123dasd` and we won't have a compile error.